packages/salesforcedx-apex-replay-debugger/src/commands/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+export enum ActionScriptEnum {
```

Comment:
Both the ActionScriptEnum and OrgInfoError had to be moved from the debugger extension (salesforcedx-vscode-apex-replay-debugger) to the adapter (salesforcedx-apex-replay-debugger) to prevent a circular reference. The extension can pull things from the adapter but not the reverse.